resources/public/api/conf/1.0/scenarios/retrieveBusinessDetails.md

```diff
@@ -9,7 +9,44 @@
     <tbody>
         <tr>
             <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulate a successful response.</p></td>
+            <td><p>Simulate a successful response with a self-employment business.
+            <p>Use the following businessId(s) for this scenario:</p>
+            <ul>
+              <li>XAIS123456789012</li>
+            </ul>
+            </p>
+            </td>
+        </tr>
+        <tr>
+            <td><p>PROPERTY</p></td>
+            <td><p>Simulate a successful response with a uk-property business.
+            <p>Use the following businessId(s) for this scenario:</p>
+            <ul>
+              <li>XAIS123456789013</li>
+            </ul>
+            </p>
+            </td>
+        </tr>
+        <tr>
+            <td><p>BUSINESS_AND_PROPERTY</p></td>
+            <td><p>Simulate a successful response with both a self-employment business and a uk-property business.
+            <p>Use the following businessId(s) for this scenario:</p>
```

Comment:
`businessId(s)` -> `businessIds`

Reply:
In fact, there's no real reason to include this scenario - it can probably be removed (on this endpoint it basically duplicates the above two scenarios)